docs/user/boot_options/index.md

```diff
@@ -38,6 +38,17 @@ first one can be removed anytime.
   inst.config_url=usb:///agama.yaml
   ```
 
+- `inst.dud`
+  Allows to patch or extend the Live Media by ponting to so called Driver Update. It currently
+  supports only RPM packages. See more about supported URLs at [URLs section](/docs/user/urls).
+  No checks of dependencies and no validation is currently provided. Multiple entries are allowed,
+  but each of them is handled separately on its own.
+
+```text
+inst.dud=https://download.opensuse.org/some-project/some.rpm
+inst.dud=label://OEMDRV/package.rpm
```

Comment:
OEMDRV is not supported yet, right? In that case I'd omit that to avoid confusion.

Reply:
No, this is a specific label specified in the URL, that one is supported.
Automated search for OEMDRV is not supported.